client/analysisui/rmanovafactorscontrol.js

```diff
@@ -60,13 +63,83 @@ var rmafcItem = function(parent, data, isFirst, isLast) {
 
         if (isEmpty === false) {
             for (var i = 0; i <= levels.length; i++)
-                this.createLevel(levels[i], i);
+                this.createLevel(index, levels[i], i);
+        }
+    };
+
+    this.intToRoman = function(number) {
+        let x = number;
+        let diff = 0;
+        let count = 0;
+        let value = '';
+        let roman = [];
+
+        if (number > 3999)
+            throw "Can not convert to roman numeral. Number to large.";
+
+        do {
+            if (count === 0)
+                roman = ['I', 'II', 'III', 'IV', 'V', 'VI', 'VII', 'VIII', 'IX'];
+            else if (count === 1)
+                roman = ['X', 'XX', 'XXX', 'XL', 'L', 'LX', 'LXX', 'LXX', 'XC'];
+            else if (count === 2)
+                roman = ['C', 'CC', 'CCC', 'CD', 'D', 'DC', 'DCC', 'DCCC', 'CM'];
+            else if (count === 3)
+                roman = ['M', 'MM', 'MMM'];
+
+            diff = x % 10;
+            if (diff > 0) {
+                value = roman[diff - 1] + value;
+                x -= diff;
+            }
+            x /= 10;
+            count += 1;
+        }
+        while (x >= 1);
+
+        return value;
+    };
+
+    this.getSequenceChar = function(seq, index) {
+        seq = seq % 4;
+        let alph = [];
+        if (seq === 0)
+            return (index + 1).toString();
+        else if (seq === 1)
+            return this.intToRoman(index + 1);
```

Comment:
i think roman numerals are probably the least clear `I` looks like `1` for example, maybe put them 4th, after letters and greek letters?